pom.xml

```diff
@@ -660,7 +660,7 @@
 				<plugin>
 					<groupId>org.codehaus.mojo</groupId>
 					<artifactId>build-helper-maven-plugin</artifactId>
-					<version>3.6.0</version>
```

Comment:
This change does not look related to the ticket.

Reply:
It's not related to this ticket, it was changed by a different branch.